tests/test_the_simplest_thing.py

```diff
@@ -0,0 +1,9 @@
+
+def test_true_is_not_false():
```

Comment:
I would like a test case that actually imports something from the `gateway` library. If the package structure inside `./gateway` dir was still broken, chances are these would still succeed.

2 things to test:

- [ ] import this SQLAlchemy database thingy and assert it's just that, i.e. `test_db_is_a_sqlalchemy_database` test case with a `assert isinstance(db, SQLAlchemy)`
- [ ] import the *FirebaseId* class from `./gateway/resources/firebase.py` and have a test like `test_firebase_id_is_a_flask_restful_resource` (`assert issubclass(FirebaseId, flask_restful.Resource)`)

Reply:
Done!